dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -448,6 +443,7 @@ static void runInParallel(
         if (part_result.error == ReadFromStreamError::ErrUpdateSchema)
         {
             prehandle_task->abort_flag.store(true);
+            prehandle_task->abort_error.store(ReadFromStreamError::ErrUpdateSchema);
```

Comment:
I think this fix could lead to the same issue again:

Thread A: run `prehandle_task->abort_flag.store(true)` and CPU yield before `prehandle_task->abort_error.store(ReadFromStreamError::ErrUpdateSchema);`
Thread B: read `prehandle_task->abort_error.load()` and see the reason is "abort"
Thread A: resume and set the reason is "update schema"

Reply:
fixed by using a single atomic variable